src/constants.js

```diff
@@ -26,3 +27,6 @@ export const PRESETS = [
     type: 'gerrit',
   },
 ];
+
+// Google Analytics Tracking ID.
```

Comment:
Optional: How about making this to the Jsdoc style? 

/** Google analytics Tracking ID */